frontend/Dockerfile

```diff
@@ -108,6 +108,10 @@ ARG SENDY_API_URL
 ENV SENDY_API_URL=${SENDY_API_URL}
 ARG SENDY_LIST_ID
 ENV SENDY_LIST_ID=${SENDY_LIST_ID}
+ARG API_AUTH_TOKEN
+ENV API_AUTH_TOKEN=${API_AUTH_TOKEN}
+ARG API_URL
+ENV API_URL=${API_URL}
```

Comment:
FYI, to the best of my knowledge, this isn't necessary. Feel free to keep in here though.